testing/cli-e2etest/environment/jaeger/resources/data-store.yaml

```diff
@@ -4,7 +4,6 @@ spec:
   name: jaeger
   type: jaeger
   default: true
-  createdAt: 2023-05-09T18:22:34.840021Z
```

Comment:
This, and all similar updates within the cli e2e yaml files are because the new apply implementation doesn't override the file. it just injects the ID if needed. This means 2 things;

1. no more commits to these files because of running tests locally and changing create dates,
2. the original file format is now preserved, including comments and other user generated formattings